toolkit/committee-selection/pallet/src/lib.rs

```diff
@@ -12,8 +13,8 @@ pub mod session_manager;
 
 pub use pallet::*;
 
-#[cfg(any(test, feature = "mock"))]
-pub mod mock;
+#[cfg(test)]
+mod mock;
```

Comment:
This deserves a line in the changelog. I understand that chance of it being used is slim.